test/python/test_pack_e2e.py

```diff
@@ -0,0 +1,197 @@
+import pytest
+import os
+import tarfile
+import subprocess
+
+from utils import basepath
+from utils import examine_application_instance_container
+from utils import tarantool_enterprise_is_used
+from utils import Image, find_image, delete_image
+from utils import Archive, find_archive
+from utils import tarantool_repo_version
+
+
+# #######################
+# Class InstanceContainer
+# #######################
+class InstanceContainer:
+    def __init__(self, container, instance_name, http_port, advertise_port):
+        self.container = container
+        self.instance_name = instance_name
+        self.http_port = http_port
+        self.advertise_port = advertise_port
+
+
+# ########
+# Fixtures
+# ########
+@pytest.fixture(scope="function")
+def image_name_for_tests(docker_client, tmpdir, request):
+    if tarantool_enterprise_is_used():
```

Comment:
It could a fixture with scope state. In fact simple boolean value

Reply:
I don't like the idea to pass it explicitly in every test where it's used.
I sure that using lazy initialization for `tarantool_version` is enough to use this function without additional optimization.